src/controllers/applicationsContentController.js

```diff
@@ -0,0 +1,304 @@
+        templateContent = {
+            applicationsMetadata: metaData,
+            baseUrl: constants.BASE_URL + config.port
+        }
+    }
+    else {
+        console.log('/' + orgName);
+        metaData = await getAPIMApplications();
+        templateContent = {
+            applicationsMetadata: metaData,
+            baseUrl: '/' + orgName
+        }
+    }
+    html = renderTemplate('../pages/applications/page.hbs', filePrefix + 'layout/main.hbs', templateContent, true);
+    res.send(html);
+}
+
+async function getMockApplications() {
+    const mockApplicationsMetaDataPath = path.join(process.cwd(), filePrefix + '../mock/Applications', 'applications.json');
+    const mockApplicationsMetaData = JSON.parse(fs.readFileSync(mockApplicationsMetaDataPath, 'utf-8'));
+    return mockApplicationsMetaData.list;
+}
+
+async function getAPIMApplications() {
+    try {
+        const response = await axios({
+            method: 'GET',
+            url: controlPlaneUrl + '/applications',
+            headers: {
+                Authorization: `Bearer ${token}`,
+            },
+            httpsAgent
+        });
+        return response.data.list;
+    } catch (error) {
+        console.error('Error fetching applications:', error.message);
+    }
+}
+
+// ***** Load Throttling Policies *****
+
+const loadThrottlingPolicies = async (req, res) => {
+    const orgName = req.params.orgName;
+    let html, metaData, templateContent;
+    if (config.mode === constants.DEV_MODE) {
+        metaData = await getMockThrottlingPolicies();
+        templateContent = {
+            throttlingPoliciesMetadata: metaData,
+            baseUrl: constants.BASE_URL + config.port
+        }
+    }
+    else {
+        metaData = await getAPIMThrottlingPolicies();
+        templateContent = {
+            throttlingPoliciesMetadata: metaData,
+            baseUrl: '/' + orgName
+        }
+    }
+    html = renderTemplate('../pages/add-application/page.hbs', filePrefix + 'layout/main.hbs', templateContent, true);
+    res.send(html);
+}
+
+async function getMockThrottlingPolicies() {
+    const mockThrottlingPoliciesMetaDataPath = path.join(process.cwd(), filePrefix + '../mock/Applications', 'throttlingPolicies.json');
+    const mockThrottlingPoliciesMetaData = JSON.parse(fs.readFileSync(mockThrottlingPoliciesMetaDataPath, 'utf-8'));
+    return mockThrottlingPoliciesMetaData.list;
+}
+
+async function getAPIMThrottlingPolicies() {
+    try {
+        const response = await axios({
+            method: 'GET',
+            url: controlPlaneUrl + '/throttling-policies/application',
+            headers: {
+                Authorization: `Bearer ${token}`,
+            },
+            httpsAgent
+        });
+        return response.data.list;
+    } catch (error) {
+        console.error('Error fetching throttling policies:', error.message);
+    }
+}
+
+// ***** Load Application *****
+
+const loadApplication = async (req, res) => {
```

Comment:
why do we need two methods loadApplication and loadApplicationForEdit?

Reply:
loadApplicationForEdit is being used inside loadApplication for one flow. This is to improve readability